inc/floppy.h

```diff
@@ -54,7 +54,7 @@ struct hfe_image {
     uint16_t tlut_base;
     uint16_t trk_off;
     uint16_t trk_pos, trk_len;
-    bool_t is_v3, double_step;
+    bool_t is_v3, double_step, fresh_seek;
```

Comment:
Not really a seek but a fresh write. And I think you can `hfe.trk_pos == hfe.write.start` instead and avoid the boolean altogether. This would possibly trigger if the write wraps round the whole track, but harmlessly so I think.

Reply:
Ah. `hfe.write` was deleted in the v4 branch, so that wasn't an option. I'll make that change, and add the boolean back in the v4 branch.